src/dashboards.js

```diff
@@ -13,14 +13,24 @@ const superagent = require('superagent');
 const CONFIG_PATH = path.join(__dirname, "config.json");
 
 
+<<<<<<< HEAD
```

Comment:
merge conflict 😂

Reply:
I was trying to rebase and messed up. I hard reset to the previous commit I had